js_modules/dagster-ui/packages/ui-core/src/runs/Run.tsx

```diff
@@ -346,3 +349,23 @@ const LogsContainer = styled.div`
   flex-direction: column;
   height: 100%;
 `;
+
+const NoStepSelectionState = ({type}: {type: LogType}) => {
+  return (
+    <Box
+      flex={{
+        direction: 'row',
+        grow: 1,
+        alignItems: 'center',
+        justifyContent: 'center',
+      }}
+      style={{background: '#221F1B'}}
+    >
+      <NonIdealState
+        title={`Select a step to view ${type.toUpperCase()}`}
+        icon="warning"
+        description="Select a step from the dropdown above or by clicking directly on a step in the chart."
```

Comment:
This might be the language in the designs but I'd vote for `Select a step on the Gantt chart or from the dropdown above to view logs.`